content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Add / edit attribute value
+weight: 1
+---
+
+# Delivery slips
+
+In this page 2 sections are present:
+
+## Print PDF
+
+2 Input fields are presented:
+
+1) From
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+2) To
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+### Call to action
+
+ - Generate PDF, when pressed, if no PDF was done on the selected time, will show the message: "No delivery slip was found for this period."
+
+## Delivery slip options
+
+3 Input felds are presented:
+
+1) Delivery prefix
+
+This is an input field for the delivery prefix, depending on the countries installed on the shop there will be am input ready in the field, if for example the default country in the shop is Germany, the input will be: "#DE". Has a tooltip: "Prefix used for delivery slips".
```

Comment:
How does it works with multi-language BO ? 

The behavior looks weird to me (1.7.8.0)

For example 
- if i choose fr (french) I get "#LI" prefix 
- if I choose gb I get "#DE" prefix

Reply:
![image](https://user-images.githubusercontent.com/75423142/167099556-acef3a2e-0511-4401-b26f-ffe00d2e7d96.png)


My prefix does not change, should this be registered as a bug? It stays on DE even when switching. PS 1784 version